src/seedu/addressbook/AddressBook.java

```diff
@@ -98,40 +98,40 @@
     private static final String PERSON_STRING_REPRESENTATION = "%1$s " // name
                                                             + PERSON_DATA_PREFIX_PHONE + "%2$s " // phone
                                                             + PERSON_DATA_PREFIX_EMAIL + "%3$s"; // email
-    private static final String COMMAND_ADD_WORD = "add";
+    private static final String ADD = "add";
```

Comment:
The precious name would be preferred. It provides more details about the variable.